flake.nix

```diff
@@ -209,6 +213,15 @@ rec {
 
             npmConfigHook = pkgs.importNpmLock.npmConfigHook;
 
+            makeCacheWritable = true;
+
+            preConfigure = ''
+              mkdir -p @NixOS/branding
+              cp -R --no-preserve=mode,ownership ${
+                branding.hydraJobs.nixos-branding.${system}.npm-package
```

Comment:
Rather than accessing `npm-package` directly through the flake input, wouldn't it be better to use the expose default overlay? This would add `nixos-branding` to the top-level of the import package set and then you could access `npm-package` via `nixos-branding.npm-package`.